docs/getting-started/index.md

```diff
@@ -58,17 +59,13 @@ stack {
 }
 ```
 
-Any file with the extension `.tm` or `.tm.hcl` is recognized by Terramate and any Terramate file that contains a `stack {}` block is recognized as a stack.
-
-A Terramate Stack is a directory that contains Terramate configuration files. This is where Terramate will generate Terraform. A stack has a 1:1 relationship with the Terraform state - i.e. each stack is a directory where you can run `terraform init`
-
-Moreover, there is nothing special about this `stack.tm.hcl` file and it can be generated manually without the ID or the `terramate create` command.
+Terramate recognizes files with the `.tm` or `.tm.hcl` extension, and any file containing a `stack {}` block is considered a stack. A stack is simply a directory where Terramate generates Terraform files. 
 
-However, it’s preferable to generate unique IDs for our stacks because, as we build more complex infrastructures, they allow us to reference stacks directly rather than using relative paths, which makes refactoring painful.
+You can create the `stack.tm.hcl` file manually without an ID or by running the `terramate create` command, but generating unique IDs for stacks is recommended. Unique IDs enable direct stack referencing instead of using relative paths, which can make refactoring difficult.
```

Comment:
```suggestion
> **Note:** You can manually create the `stack.tm.hcl` file without an ID or by running the `terramate create` command.
> However, generating unique IDs for stacks is recommended since they enable direct stack referencing instead of relative paths, making refactoring difficult.
```

I would add this as a quote or so and make sure that the user understands it's optional but not the desired way

Reply:
Yes, this makes sense. Get readers' attention and conveys the message easily.